scripts/model-push/llama-converter/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ignaciolopezluna020/llama.cpp:cpu
+FROM ghcr.io/ggml-org/llama.cpp:full-b5227
```

Comment:
Maybe there is a smaller image that would also work, but did not find the `cpu` tag in GHCR.